### DISCUSSION TITLE
Voxel support

### DISCUSSION THREAD

**Discussion by vault-thirteen** [o3de/o3de#17117] [Help Wanted]
Is voxel support planned for any foreseeable future ?

**Comment by ceceomer** [comment]
What do you mean? A voxel is cube mesh. So what do you want to do?

**Comment by vault-thirteen** [comment]
Voxels can be used in many ways. The most obvious one of them is a modifiable terrain such as in the `No Man's Sky` computer game. The simpliest video example is following – https://www.youtube.com/watch?v=y3m47RPxt8Q . More advanced types of usage may be seen in modifyable objects, such as destroyable objects, and voxel-based physics engine which simulates real life physics. Simple example is here – https://www.youtube.com/watch?v=ddsvqgcGvR0 .

**Reply by ceceomer** [reply]
"modifyable objects, such as destroyable objects, and voxel-based physics" all of them is about your code. there is no general porpose game engine for them. I did minecraft clone in Unity3D and all of them are written by me. It is pointless to expect this kind of support.

**Comment by vault-thirteen** [comment]
> It is pointless to expect this kind of support.

Are you a developer of the `O3DE` ?

P. S.

William Henry Gates III, also known as Bill Gates, once said that 640 kilobytes of memory (RAM) would be sufficient for everything. Looking at how many gigabytes, not kilobytes, of RAM are present in modern computers, it is evident that the world has changed.
